package.json

```diff
@@ -31,9 +31,7 @@
     "prepare": "husky install",
     "app:dev": "npx concurrently -k -n 'API,APP,ALERTS-TASK,COMMON-UTILS' -c 'green.bold,blue.bold,yellow.bold,magenta' 'nx run @hyperdx/api:dev' 'nx run @hyperdx/app:dev' 'nx run @hyperdx/api:dev-task check-alerts' 'nx run @hyperdx/common-utils:dev'",
     "app:lint": "nx run @hyperdx/app:ci:lint",
-    "predev": "docker compose -f docker-compose.dev.yml up -d",
-    "dev": "yarn app:dev",
-    "postdev": "docker compose -f docker-compose.dev.yml down",
```

Comment:
since custom post/pre scripts are not supported in yarn, merge them into dev script